src/rust.rs

```diff
@@ -1802,3 +1802,23 @@ pub mod default_on_null {
         value.serialize(serializer)
     }
 }
+
+/// Deserialize any value, ignore it, and return the default value for the type being deserialized
```

Comment:
Could you please extend the description with a note that this only works for enums with data.
And maybe also the part about simulating an other-like variant in untagged enums. Basically the info in my last comment in #250.

Reply:
I added a description giving more context about when and why the function should be used.